lib/public/Model.js

```diff
@@ -59,6 +59,8 @@ export default class Model extends Observable {
                 this.logs.fetchAllLogs();
                 break;
             case 'entry':
+                this.logs.data = [];
+                this.logs.filteredData = [];
```

Comment:
Why initiate the arrays in the route switch case?

Reply:
Some old code, has been removed.